src/Form/SettingsForm.php

```diff
@@ -144,7 +144,8 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
     $config->save();
     parent::submitForm($form, $form_state);
     $message = Link::createFromRoute($this->t('View events list'), 'event_database_pull.events_list')->toString();
-    drupal_set_message($message);
+    // @todo use dependency injection.
+    \Drupal\Core\Messenger\MessengerInterface::addMessage($message);
```

Comment:
This will not work: You cannot call a function on an interface. Use dependency injection.